app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -130,6 +131,28 @@ def users_in_group(name)
     ldap_con.user_list(name)
   end
 
+  def test_connection
+    result = {}
+    begin
+      Timeout::timeout(10) do
```

Comment:
Either make this a setting - or make it much larger, 60s even. Not everyone has a fast connection :smile: